src/tech/data-standards.md

```diff
@@ -0,0 +1,47 @@
+
+These [GSS harmonised standards and their accompanying guidance](https://analysisfunction.civilservice.gov.uk/government-statistical-service-and-statistician-group/gss-support/gss-harmonisation-support/harmonised-standards-and-guidance/) should accordingly be our first point of reference when planning data collection, rationalising data stores, or creating data mappings. The standards cover a wide range of subjects, from [fear of crime](https://analysisfunction.civilservice.gov.uk/policy-store/crime-and-fear-of-crime/) to [social-capital metrics](https://analysisfunction.civilservice.gov.uk/policy-store/social-capital/), and are potentially applicable to a very wide range of use-cases. 
+
+### Data Standards Authority endorsed standards
+
+The Data Standards Authority reviews standards and controlled vocabularies for use by central government. The Authority maintains a [list of standards](https://alphagov.github.io/data-standards-authority/standards/) they consider valuable for UK government use. These should be applied where relevant.
+
+## When not to use standards
+
+All standards mandated or recommended by the UK Government have been extensively tested and their value proven 'in the wild'. Under most circumstances, then, their use provides an easy and effective means of improving data collection for our clients.
+
+Some services and applications, however, will have specialised or unique requirements unsuitable for standardisation. For instance, standard age-bandings may not be useful to a service focused on Early Years Foundation Stage education; here, finer-grained measures will probably be needed. Be sure to consult with relevant stakeholders before implementing a standard to confirm that it does in fact meet their needs. Even in cases where a standard as defined cannot be directly applied, it will often be possible to create data structures that can be mapped to it without information loss; for instance, in the age-banding example given above, care could be taken to ensure that the more-specific groupings defined fall within the boundaries of one of the less-granular standards.
```

Comment:
I would make this point a little more strongly. ONS/GSS/GDS standards tend to be designed for maximum clarity for the general public, whereas many of the things we build the users are internal specialists or power users for whom different standards or nomenclature might be more appropriate – for example the police use [IC codes](https://en.wikipedia.org/wiki/IC_codes) to describe ethnicities.

Reply:
I've expanded slightly to emphasise that the data standards described are 'sensible defaults' rather than 'universal solutions'. I haven't referred to the IC codes specifically, partly because the Wikipedia entry goes into some detail about the relationship of IC codes to other standards and the example started seeming less clear-cut than desirable.